5calls/app/src/main/java/org/a5calls/android/a5calls/controller/MainActivity.java

```diff
@@ -190,22 +214,15 @@ private void showStats() {
         Snackbar.make(findViewById(R.id.activity_main), message, Snackbar.LENGTH_LONG).show();
     }
 
-    private void onLocationUpdated() {
+    private void refreshIssues() {
         if (!TextUtils.isEmpty(mLatitude) && !TextUtils.isEmpty(mLongitude)) {
-            onLatLongUpdated(mLatitude, mLongitude);
-        } else if (! TextUtils.isEmpty(mZip)) {
-            onZipUpdated(mZip);
-        }
-    }
+            AppSingleton.getInstance(getApplicationContext()).getJsonController()
+                    .getIssuesForLocation(mLatitude + "," + mLongitude);
 
-    private void onZipUpdated(String zip) {
-        AppSingleton.getInstance(getApplicationContext()).getJsonController()
-                .getIssuesForLocation(zip);
-    }
-
-    private void onLatLongUpdated(String latitude, String longitude) {
-        AppSingleton.getInstance(getApplicationContext()).getJsonController()
-                .getIssuesForLocation(latitude + "," + longitude);
+        } else if (!TextUtils.isEmpty(mZip)) {
+            AppSingleton.getInstance(getApplicationContext()).getJsonController()
+                    .getIssuesForLocation(mZip);
+        }
```

Comment:
Do you want me to just add it?  What should we do in that case?  Toast the user and log an error to the backend?